### PR TITLE
Add license

### PR DESCRIPTION
**Pull request by acron0**

### CODE REVIEW

**Comment by sw1nn** [comment]
@otfrom is the expert on this, but I think for EPL you have to supply a copy of the license agreement, but then EPL may not be the right license for docs, possible should be a CC thing? http://creativecommons.org/

**Comment by otfrom** [comment]
Let's go with Creative Commons Attribution 4.0 International for docs

http://creativecommons.org/choose/

The copyright is correct except for the year.

File: README.md [hunk 1]
```diff
@@ -18,3 +18,7 @@ We’re on a mission to help you, the professionals, to build better cities. The
 Witan was built and is maintained by Mastodon C.  
 Website: mastodonc.com  
 Email: theteam@mastodonc.com
+
+Copyright © 2014 Mastodon C
```
Comment (otfrom) [inline README.md]: 2015

File: README.md [hunk 2]
```diff
@@ -18,3 +18,7 @@ We’re on a mission to help you, the professionals, to build better cities. The
 Witan was built and is maintained by Mastodon C.  
 Website: mastodonc.com  
 Email: theteam@mastodonc.com
+
+Copyright © 2015 Mastodon C
```
Comment (otfrom) [inline README.md]: Mastodon C Ltd

(sorry, I always forget the Ltd)

**Comment by otfrom** [comment]
Ltd then :shipit:

**Pull request merged**